Common/src/Utils/LocalIPv4.cs

```diff
@@ -57,6 +71,12 @@ public static string GetLocalIPv4(NetworkInterfaceType _type)
     return output;
   }
 
+  /// <summary>
+  ///   Get local IP from Ethernet network interface.
```

Comment:
Should we add that we identify the owner pod with this value?

Reply:
Not in this class. This is a general usage class that may not be linked to our usage of it.

Reply:
Ok